dir_class.py

```diff
@@ -63,18 +66,25 @@ def get_tags(self):
                 album_artist = audio["albumartist"][0]
             except KeyError:
                 album_artist = ""
-        tags = {"album_title": album_title, "album_artist": album_artist}
-        cos = {}
-        selected = [0, "whoever"]
-        for x in tags.values():
-            if x == "":
+            if album_artist == "":
                 continue
```

Comment:
Tu będzie potrzebna nieco grubsza zmiana w logice działania.

Jeżeli nie będziesz miała podanego artysty albumu, to continue spowoduje, że przejdziesz do kolejnej iteracji pętli i nie wykonasz kodu dotyczącego `album_title`.